py3status/modules/coin_market.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data from https://coinmarketcap.com
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=#ff0 {name}] ${price_usd:.2f} {percent_change_24h}')
```

Comment:
I think it'd be nice to import 16 colors from `~/.Xresources`. It'd be `color=3` for yellow, then override colors from `color_*` inside the modules by just `color_3 = #75507b`. No giving names like `degraded` because it's not degraded here... and `color_yellow = #75507b` wouldn't make sense either (it's purple hex color).
```
format_coin = [\?color=3 {name}] ${price_usd:.2f} {percent_change_24h}
color3 = '#75507b'
```

`rofi` do something like this too in `~/.Xresources`.
```
! breeze-dark
rofi.color-enabled: true
rofi.color-window: #31363B, #002b36, #31363B
rofi.color-normal: #31363B, #ffffff, #31363B, #ffffff, #31363B
rofi.color-active: #31363B, #268bd2, #31363B, #268bd2, #ffffff
rofi.color-urgent: #31363B
```
If no color in `~/.Xresources`, we use default `linux/tango` color theme.